### PR TITLE
a few minor changes and reminders

### PR DESCRIPTION
**Pull request by cyrush**
makes a few minor tweaks and adds example host config for ornl summit

### CODE REVIEW

**Comment by cyrush** [comment]
Another question I had, why do we need this for windows?

https://github.com/LLNL/blt/blob/2b3340e9ea30e0e4a9ebcb5f6b27bf25631dd3ba/docs/tutorial/calc_pi
/CMakeLists.txt#L41

File: docs/tutorial/creating_execs_and_libs.rst [hunk 1]
```diff
@@ -125,6 +125,15 @@ the linker removing unused symbols in the larger library.
...
 
+..
+.. ------------------------------------
+.. CYRUS NOTE: 
+.. Object Libraries CUDA Considerations
+.. this seems like a detour given we haven't talked about CUDA yet
+.. ------------------------------------
+..
+
+
```
Comment (cyrush) [inline docs/tutorial/creating_execs_and_libs.rst:136]: this should be in a appendix, or maybe a rider post cuda intro

**Comment by davidbeckingsale** [comment]
> Another question I had, why do we need this for windows?
> 
> https://github.com/LLNL/blt/blob/2b3340e9ea30e0e4a9ebcb5f6b27bf25631dd3ba/docs/tutorial/calc_pi/CMakeLists.txt#L41

Looks like for Windows DLL stuff: https://github.com/LLNL/blt/blob/2b3340e9ea30e0e4a9ebcb5f6b27bf25631dd3ba/docs/tutorial/calc_pi/calc_pi_exports.h#L11

**Comment by cyrush** [comment]
@davidbeckingsale how could I forget :-)

**Review by white238 — APPROVED** [review]

**Pull request merged**